Package.swift

```diff
@@ -101,7 +101,19 @@ let package = Package(
             dependencies: ["Commands"]),
         Target(
             /** Shim tool to find test names on OS X */
-            name: "swiftpm-xctest-helper"),
```

Comment:
Turns out it was just luck that this wasn't being picked up by bootstrap script to be built in stage one 😅